lib/DBIx/Class/Helper/ResultSet/Shortcut/ResultsExist.pm

```diff
@@ -24,6 +24,9 @@ sub results_exist_as_query {
 sub results_exist {
    my $self = shift;
 
+   my $query = $self->results_exist_as_query;
+   $$query->[0] .= 'as _existence_subq';
```

Comment:
That won't work - you need to actually use `$query` further down around line 33ish

Also capitalizing `AS` will make things more inline with how DBIC/SQLAC themselves output keywords.

Reply:
sorry, that's what happens when I PR before my morning cofee.

Reply:
now it should actually be it